db/migrate/20240729103525_update_proxy_byos_column_type.rb

```diff
@@ -1,7 +1,6 @@
 class UpdateProxyByosColumnType < ActiveRecord::Migration[6.1]
   def up
-    add_column :systems, :proxy_byos_mode, :integer, default: 0
-    System.where(proxy_byos: false).in_batches.update_all proxy_byos_mode: 0
+    add_column :systems, :proxy_byos_mode, :integer, default: nil
```

Comment:
Why use `nil` as the default? This causes a different behavior than `System#proxy_byos` had, where `default: false` means the default value is PAYG.

IMO since the enum for `System#byos_proxy_mode` defines `payg: 0`, then the default _should be_ `0`.

Reply:
OK, will change from `nil` to 0, same meaning